docs/index.md

```diff
@@ -8,8 +8,44 @@
     <a class="button" href="schedule">GTFS Schedule</a><a class="button" href="realtime">GTFS Realtime</a><a class="button" href="resources">Resources</a><a class="button" href="extensions">Extensions</a>
 </div>
 
-The General Transit Feed Specification (GTFS) is a data specification that allows public transit agencies to publish their transit data in a format that can be consumed by a [wide variety of software applications](resources/apps). Today, the GTFS data format is used by thousands of public transport providers.
+The General Transit Feed Specification (GTFS) is an [Open Standard](https://www.interoperablemobility.org/definitions/#open_standard) used to distribute relevant information about transit systems to riders. It allows public transit agencies to publish their transit data in a format that can be consumed by a wide variety of software applications. Today, the GTFS data format is used by thousands of public transport providers.
 
-GTFS is split into a [schedule component](schedule) that contains schedule, fare, and geographic transit information and a [real-time component](realtime) that contains arrival predictions, vehicle positions and service advisories.
+GTFS is simple. Information about routes, schedules, and fares are represented in basic text files, which means it can be created and maintained without the use of complicated or proprietary software. It is split into a schedule component that contains schedule, fare, and geographic transit information and a real-time component that contains arrival predictions, vehicle positions and service advisories.
```

Comment:
This section now sounds as if GTFS-Realtime consists of basic text files too.

Reply:
Good catch! Perhaps an improved edit: 

> GTFS is split into a schedule component that contains schedule, fare, and geographic transit information and a real-time component that contains arrival predictions, vehicle positions and service advisories. GTFS Schedule is simple. Information about routes, schedules, and fares are represented in basic text files, which means it can be created and maintained without the use of complicated or proprietary software.